modules/metastorage-common/src/main/java/org/apache/ignite/metastorage/common/OperationTimeoutException.java

```diff
@@ -0,0 +1,59 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.ignite.metastorage.common;
+
+/**
+ * Thrown when an operation is not executed within a specified time period. Usually in such cases the operation
+ * should be retried.
+ */
+public class OperationTimeoutException extends RuntimeException {
```

Comment:
I suggest to get rid of this exception.
We already have well known java.util.concurrent.TimeoutException for this purpose.

Reply:
It is definitely different exceptions.

`java.util.concurrent.TimeoutException` will be thrown on timed waiting on, for example, `future.get(timeout)` operation, while this `timeout` is not related with service timeout yet.